### PR TITLE
fix: Workaround absolute position issue

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): WORKAROUND #6964 (not sure if it's good enough). Would love to fix the root issue if someone can provide some help :)

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Manipulation event args position is always absolute (relative to screen)

## What is the new behavior?

It's now relative to the control as UWP, except for when the app isn't at the same position as screen.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Input/Internal/ManipulationEventArgsHelpers.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using Windows.UI.Xaml;
+
+namespace Uno.UI.UI.Xaml.Input.Internal
+{
+	internal static class ManipulationEventArgsHelpers
+	{
+		internal static Point MapPointRelativeTo(UIElement element, Point point)
+			=> element is null
+				? point
+				: element.TransformToVisual(null).Inverse.TransformPoint(point);
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Input/Internal/ManipulationEventArgsHelpers.cs:11]: Stolen from @Xiaoy312 sample :) Thanks for the workaround!

Comment (dr1rrb) [reply src/Uno.UI/UI/Xaml/Input/Internal/ManipulationEventArgsHelpers.cs:11]: I'm really not a fan of this one: `TransformToVisual(null)` is really intense as it required to walk the whole tree. Depending of the platform we can have an optimized way to get relative coordinates from the `PointerRoutedEventArgs` that are not so far from here ...

Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Input/Internal/ManipulationEventArgsHelpers.cs:11]: Let me close the PR for now. I wasn't able to get it, and the workaround can easily be integrated into apps that need it. So no need for such a workaround in Uno itself.

**Pull request closed without merging**